### ISSUE TITLE
Some problems of flow_warp.py

### ISSUE THREAD

**Issue by Shuweis** [open-mmlab/mmediting#638]
I saw the flow_warp.py and was confused of the code annotation （w,h,2）：
```
 grid_y, grid_x = torch.meshgrid(torch.arange(0, h), torch.arange(0, w))
 grid = torch.stack((grid_x, grid_y), 2).type_as(x)  # (w, h, 2)
```
I try to see the dimension of grid and got （h,w,2）.Is there something wrong?

**Comment by ckkelvinchan** [comment]
`(h, w, 2)` is correct. It is the comment that is wrong. Thanks for letting us know.

**Comment by ckkelvinchan** [comment]
Fixed in #655, thanks!
